dist/tools/openocd/openocd.sh

```diff
@@ -119,6 +120,11 @@
 # export DBG_FLAGS='--debugger "${GDB} ${DBG_DEFAULT_FLAGS}"'
 # The single quotes are important on the line above, or it will not work.
 
+# Set extra commands to pass to OpenOCD before `flash` or `reset`
```

Comment:
This comment should be updated now I guess to match the `connect_assert_srst` behavior only.

The documentation could even be put at the beginning with the other configuration variables by setting it to empty by default and here just "handle OPENOCD_RESET_USE_CONNECT_ASSERT_SRST".

Reply:
Agree, is the following ok?

```
# If set to 1 'reset_config' will use 'connect_assert_srst' before 'flash' or 'reset.
: ${OPENOCD_RESET_USE_CONNECT_ASSERT_SRST:=}
```

Reply:
Perfect